Add possibility for extra envs

pixiono opened the pull request:
Since we have a cluster in an environment without Internet access, we needed a proxy to reach Azure AD. For this we need appropriate environment variables.

ubergesundheit commented:
Hello @pixiono could you also update the CHANGELOG.md file https://github.com/giantswarm/oauth2-proxy-app/blob/main/CHANGELOG.md ?

Did you test your changes with multiple entries and add an example in the `values.yaml` file as a comment?

ubergesundheit commented:
Please also update the `values.schema.json` file next to the values.yaml. Thanks!

pixiono commented:
@ubergesundheit thanks for your reply, I added the requested information. 

>  you test your changes with multiple entries and add an example in the values.yaml file as a comment?
Yes, I did. I also added the example in the values.yaml

pixiono commented on CHANGELOG.md:
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.8.0] - 2023-02-01
```
[Done](https://github.com/giantswarm/oauth2-proxy-app/pull/103/commits/39defa1f98968f139dacaaa0056ce80d3582a025)

pixiono commented on CHANGELOG.md:
```diff
@@ -143,7 +153,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Push `g8s-oauth2-proxy` app CRs into `<provider>-app-collection` repository.
 
-[Unreleased]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.7.1...HEAD
+[Unreleased]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.8.0...HEAD
```
[Done](https://github.com/giantswarm/oauth2-proxy-app/pull/103/commits/39defa1f98968f139dacaaa0056ce80d3582a025)

pixiono commented on helm/oauth2-proxy/Chart.yaml:
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: oauth2-proxy
 description: A reverse proxy that provides authentication with Google, Azure, OpenID Connect and many more identity providers
 home: https://github.com/giantswarm/oauth2-proxy-app
-version: "2.7.1"
```
[Done](https://github.com/giantswarm/oauth2-proxy-app/pull/103/commits/39defa1f98968f139dacaaa0056ce80d3582a025)

ubergesundheit commented on CHANGELOG.md:
```diff
@@ -144,6 +153,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Push `g8s-oauth2-proxy` app CRs into `<provider>-app-collection` repository.
 
 [Unreleased]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.7.1...HEAD
+[2.8.0]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.7.1...v2.8.0
```
```suggestion
```

ubergesundheit commented:
Thank you for your contribution!

ubergesundheit commented:
We just released your contribution as https://github.com/giantswarm/oauth2-proxy-app/releases/tag/v2.8.0

it should be available for installation soon
